src/main/scala/com/ironcorelabs/IronSdkSync.scala

```diff
@@ -44,13 +62,90 @@ case class IronSdkSync[F[_]](underlying: jsdk.IronSdk)(implicit syncF: Sync[F])
     } yield DocumentEncryptResult(result)
 
   def documentDecrypt(encryptedBytes: ByteVector): F[DocumentDecryptResult] =
-    syncF.delay(underlying.documentDecrypt(encryptedBytes.toArray)).map(DocumentDecryptResult.apply)
+    for {
+      result <- syncF.delay(underlying.documentDecrypt(encryptedBytes.toArray))
```

Comment:
IMO, any single line for should just be written as `syncF.delay(underlying.documentDecrypt(encryptedBytes.toArray)).map(DocumentDecryptResult)`, but if you prefer the other, I won't force the issue.

Reply:
I liked the consistency of the `for`s, but I also like being more concise, so I changed over all the single-line `for`s